openadapt/adapters/som.py

```diff
@@ -0,0 +1,66 @@
+import fire
+from gradio_client import Client
+from loguru import logger
+from PIL import Image
+import tempfile
+import typing
+
+
+# Function to save PIL.Image to a temporary file and return the file path
+def save_image_to_temp_file(img: Image.Image) -> str:
+    temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".png")
+    img.save(temp_file, format="PNG")
+    temp_file.close()  # Important to close the file so others can access it
+    logger.info(f"Image saved to temporary file: {temp_file.name}")
+    return temp_file.name
+
+
+def predict(server_url: str = "http://44.222.184.95:6092/",
```

Comment:
Can you please modify this to default to `config.DEFAULT_SOM_SERVER_URL` and add that to `config.py` and `.env`?